pkg/mock/fileUtils.go

```diff
@@ -125,20 +189,29 @@ func (f *FilesMock) DirExists(path string) (bool, error) {
 // Copy checks if content has been associated with the given src path, and if so copies it under the given path dst.
 func (f *FilesMock) Copy(src, dst string) (int64, error) {
 	f.init()
-	content, exists := f.files[f.toAbsPath(src)]
-	if !exists || content == &dirContent {
-		return 0, fmt.Errorf("cannot copy '%s': %w", src, os.ErrNotExist)
```

Comment:
Wrapping os.ErrNotExist was important. Calling code may check this and behave differently.

Reply:
That is what `%w` does. It's just an alternative to `errors.Wrapf()` giving more control over where to embed the error string.

Reply:
Dont know all the details here, but isn't it better to have something like `Unwrap` which gives the error from below so that I can do a `reflect.TypeOf` and check for a certain error like that? Sounds more reliable to me compared to comparing strings.

But that here is a side-thread anyway ...

Reply:
cool ! thanks ! Something learned today ...